README.md

```diff
@@ -133,6 +133,76 @@ const myLightDashboard = presetSdk.embedDashboard({
 });
 ```
 
+### Managing the dashboard filter state
+
+By default, a dashboard is loaded in Embedded mode with its default filter configuration. It's possible to pass a `permalink_key` to load the dashboard with a particular filter configuration:
+
+```javascript
+const myLightDashboard = presetSdk.embedDashboard({
+  id: dashboardId,
+  supersetDomain: supersetDomain,
+  mountPoint: document.getElementById("dashboard-container"),
+  fetchGuestToken: async () => fetchGuestTokenFromBackend(),
+  dashboardUiConfig: {
+    urlParams: {
+      "permalink_key": "aE6zJGOJK3k" // Key generated via the API with the desired filter state
+    }
+  }
+});
+```
+
+It's also possible to retrieve the current data mask configuration (which includes the filter state) at any time using the `getDataMask()` method:
+
+``` javascript
+const dashboardElement = await myLightDashboard; // `myLightDashboard` is a promise that resolves to the dashboard instance
+...
+const currentDataMaskConfig = await dash.dashboardElement();
```

Comment:
I think there’s something wrong here? Shouldnt it be sth like
```
const dashElement = await dash;
const datamask = dashElement.getDataMask()
```

Reply:
ooof you're right -- it should be `await dashboardElement.getDataMask()`. Wrong copy/pasting